ServerCore/Pages/Teams/Play.cshtml.cs

```diff
@@ -33,6 +33,8 @@ public async Task OnGetAsync(int teamId, SortOrder? sort)
             this.TeamID = teamId;
             this.Sort = sort;
 
+            await PuzzleStateHelper.CheckForTimedUnlocksAsync(_context, Event, teamId);
```

Comment:
Does this generate a task that auto-runs or just one that runs every time the user refreshes the page?

Reply:
It runs every time the user refreshes the page - but CheckForTimedUnlocks won't do its job any more frequently than once per 10 minutes (per team).

Reply:
Got it - thanks :)